client/components/layout/MainLayout.tsx

```diff
@@ -42,6 +42,19 @@ const MainLayout: React.FC = () => {
     matching && setDrawerOpen(false);
   }, [matching, setDrawerOpen]);
 
+  const auth = getAuth();
+  const handleSignout = () => {
+    signOut(auth)
```

Comment:
Shall we use a try-catch block to be consistent with the rest of the codebase?

Reply:
@kinskiwu Done.